dissect/target/tools/info.py

```diff
@@ -0,0 +1,145 @@
+
+    if not args.targets and ("-h" in rest or "--help" in rest):
+        parser.print_help()
+        parser.exit()
+
+    process_generic_arguments(args)
+
+    if not args.targets and not args.from_file:
+        parser.error("too few arguments")
+
+    if args.from_file:
+        with open(args.from_file, "r") as f:
+            targets = f.read().splitlines()
+            while targets[-1] == "":
+                targets = targets[:-1]
+            args.targets = targets
+
+    for target in Target.open_all(args.targets):
+
+        if args.json:
+            print(json.dumps(obj_target_info(target)))
+        elif args.json_pretty:
+            print(json.dumps(obj_target_info(target), indent=4))
+        elif args.record:
+            rs = record_output(args.strings, args.json)
+            rs.write(
+                InfoRecord(
+                    os_family=target.os,
+                    os_version=target.version,
+                    ips=list(set(target.ips + target.ips_dhcp)),
+                    architecture=target.architecture,
+                    install_date=target.install_date,
+                    last_activity=target.activity,
+                    language=target.language,
+                    timezone=target.timezone,
+                    _target=target,
+                )
+            )
+        else:
+            print_target_info(target)
+
+
+def obj_target_info(target):
+    return {
+        "hostname": target.hostname,
+        "os_family": target.os,
+        "os_version": target.version,
+        "architecture": target.architecture,
+        "domain": target.domain,
+        "ips": list(set(target.ips + target.ips_dhcp)),
+        "install_date": str(target.install_date),
+        "last_activity": str(target.activity),
+        "language": target.language,
+        "timezone": target.timezone,
+        "disks": [{"type": d.__class__.__name__, "size": d.size} for d in target.disks],
+        "volumes": [{"name": v.name, "size": v.size, "fs": v.fs.__class__.__name__} for v in target.volumes],
+        "children": [{"type": c.type, "path": str(c.path)} for c in target.list_children()],
+    }
+
+
+def print_target_info(target):
+    """print target information"""
+    print("Disks")
+    for d in target.disks:
+        print(f"- {str(d)}")
+    print("\nVolumes")
+    for v in target.volumes:
+        print(f"- {str(v)}")
+
+    children = list(target.list_children())
+    if len(children) > 0:
+        print("\nChildren")
+        for c in children:
+            print(f"- <Child type='{c.type}' path='{str(c.path)}'>")
+
+    print()
+    print(f"Hostname      : {target.hostname}")
+    print(f"OS family     : {target.os}, {target._os_plugin.__name__}")
+    print(f"OS version    : {target.version}")
+    print(f"Arch          : {target.architecture}")
+    print(f"Domain        : {target.domain}")
+    print(f"IPs           : {', '.join(list(set(target.ips + target.ips_dhcp)))}")
```

Comment:
```suggestion
    print(f"IPs           : {', '.join(set(target.ips + target.ips_dhcp))}")
```
Making a list is necessary again for the other methods but not here.